### ISSUE TITLE
[1.7.10] 1.11.6 crashes on server.

### ISSUE THREAD

**Issue by AnrDaemon** [Nuchaz/BiblioCraft#452]
```
[05:47:55] [Server thread/ERROR] [FML]: Caught exception from BiblioCraft
java.lang.NoClassDefFoundError: net/minecraft/client/gui/GuiScreen
```
[crash-2017-05-22_05.47.55-server.txt](https://github.com/Nuchaz/BiblioCraft/files/1017617/crash-2017-05-22_05.47.55-server.txt)

**Comment by Nuchaz** [comment]
derp, looks like in my effort to fix a problem with the slotted book, I forgot to call to open the gui on only the client. I'll get an update out soon and get this fixed.

**Comment by AnrDaemon** [comment]
Don't worry, you're not the only one derping today. ;)

**Comment by Nuchaz** [comment]
Just released BiblioCraft v1.11.7, which fixes this.

**Comment by AnrDaemon** [comment]
Confirmed, works like a charm.
Thanks!
